src/cpp/session/modules/SessionPackages.R

```diff
@@ -1211,7 +1211,7 @@ if (identical(as.character(Sys.info()["sysname"]), "Darwin") &&
          sourceFileExtensions,
          "R" = c("r", "q", "s"),
          "src" = c("c", "cc", "cpp", "h", "hpp"),
-         "vignettes" = c("rmd", "rnw"),
+         "vignettes" = c("rmd", "rnw", "qmd"),
```

Comment:
Not sure what this code does, but since quarto vignettes are becoming a thing, I thought, why not?